app/config/collections.php

```diff
@@ -1092,7 +1092,7 @@
                 'filters' => ['subQueryMemberships'],
             ],
             [
-                '$id' => 'search',
+                '$id' => 'search', // TODO: We should probably rename this, as it will be users-facing. Maybe even remove it? But how would I search for user with mame "Matej" OR email "Matej"?
```

Comment:
we might want to remove it completely if possible 👍🏻

Reply:
What query would I use to search for matej? (in Appwrite Dashboard UI, to keep current functionality).
```
[
  Query.search("name", "Matej"),
  Query.search("email", "Matej")
]
```

This will not work, this is AND. We would need something like `Query.search(["name", "email"], "Matej")`, but that is not supported. Should we see if we can implement that?

Do we deprecate the search field for now? Or do we search only by one attribute - which one?
I don't think implementing complex searching in the current dashboard UI is a good choice.

Reply:
I would say lets remove it and we search by one attribute on the old dashboard or add basic functionality to select the attribute to search for 👍🏻